MetalPerformanceShadersProxy.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "MetalPerformanceShadersProxy"
-  s.version          = "0.1.5"
+  s.version          = "0.2.0"
   s.summary          = "A MetalPerformanceShaders proxy that allows compilation on simulators."
   s.homepage         = "https://github.com/xmartlabs/MetalPerformanceShadersProxy"
   s.license          = { type: 'MIT', file: 'LICENSE' }
```

Comment:
is the `ios.deployment_target` value below right?

Reply:
i upgraded the minimum required ios version to ios 9.

Reply:
remember to do it!